### PR TITLE
very first bits of fh-sync server APB

### PR DESCRIPTION
**Pull request by matzew**
Initial simple version, that provisions and deprovisions

What's missing:
1) Making it bindable false and create the required secret on provision. It should create the same secret on provision as is currently created during a bind request.
2) the used Openshift template should have better naming of `mongodb` and `redis`, once that is changed, it will be applied here too

I think w/ `1.` I need a bit help - not 100% sure on this

### CODE REVIEW

**Comment by matzew** [comment]
@philbrookes can you take a look  ?

File: apb.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: fh-sync-server-apb
+image: matzew/fh-sync-server-apb
```
Comment (philbrookes) [inline apb.yml]: This needs to be changed to feedhenry, and the base64 string regenerated

Comment (matzew) [reply apb.yml]: yup :-)

File: roles/provision-fh-sync-server-apb/templates/secret.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ fhsync_secret_display_name }}
+  namespace: {{ namespace }}
+stringData:
+  name: {{ fhsync_secret_display_name }}
+  type: {{ fhsync_secret_display_name }}
```
Comment (philbrookes) [inline roles/provision-fh-sync-server-apb/templates/secret.yml.j2]: I think mcp-api requires a uri value too: https://github.com/feedhenry/mcp-standalone/blob/09e0c2a44a9b7d8d7baca2c9425c6373c0ae1851/pkg/data/mobileService.go#L238

Comment (matzew) [reply roles/provision-fh-sync-server-apb/templates/secret.yml.j2]: @philbrookes ah - ok. how do I get the URL from the sync service ? is it like {{ service.route }}  ?

Comment (philbrookes) [reply roles/provision-fh-sync-server-apb/templates/secret.yml.j2]: I did it like this: https://github.com/feedhenry/keycloak-apb/blob/54014c47e9c6edb1532be6406d6fc4c1f5784f0f/roles/provision-keycloak-apb/tasks/main.yml#L53-L55

**Pull request merged**